src/modules/extras/ExtrasConfig.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import Hashids from 'hashids'
+import { WohnenOfferModel } from '@integreat-app/integreat-api-client'
+
+export const SPRUNGBRETT_ROUTE = 'SprungbrettExtra'
+export const SPRUNGBRETT_EXTRA = 'sprungbrett'
+
+export const WOHNEN_ROUTE = 'WohnenExtra'
+export const WOHNEN_EXTRA = 'wohnen'
+
+export const hash = (offer: WohnenOfferModel) =>
```

Comment:
We should not need the hashids and hash function. This is only needed in the web because an offer does not have an ID.